### DISCUSSION TITLE
How to run shell command on selection

### DISCUSSION THREAD

**Discussion by NHOrus** [sxyazi/yazi#724] [1. Q&A] [answered]
I kinda want to choose file in preview and run command on it (to set background).

Is there a way to paste copied file name into command input or a way to substitute current selection for part of command? FAQ is silent on that.

**Comment by sxyazi** [comment, marked as answer]
Sure, you can use the `shell` command, docs here https://yazi-rs.github.io/docs/configuration/keymap/#shell

**Comment by NHOrus** [comment]
But only by binding the key, not with manual input of command. Okay, thank you.

**Reply by sxyazi** [reply]
Does `;` and `:` work for you?

https://github.com/sxyazi/yazi/blob/42cbedbe42936f61170d049f1b77541a5ab66e3d/yazi-config/preset/keymap.toml#L76-L77
